lib/Screen/HomeScreen.dart

```diff
@@ -0,0 +1,168 @@
+import 'package:flutter/material.dart';
+import 'package:hatspace/Screen/VerificationScreen.dart';
+import 'package:hatspace/main.dart';
+
+class MyHomePage extends StatefulWidget {
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  bool _isButtonDisable = false;
+  String emailValue = "";
+
+  @override
+  Widget build(BuildContext context) {
+    return Theme(
+        data: ThemeData(
+            elevatedButtonTheme: ElevatedButtonThemeData(
+                style: ButtonStyle(
+                  fixedSize: MaterialStateProperty.all<Size>(
+            const Size.fromHeight(45),
+          ),
+          backgroundColor: MaterialStateProperty.all<Color>(
+              const Color.fromRGBO(255, 255, 255, 1)),
+            ))),
```

Comment:
use this `MaterialStateProperty` to set different color for different button state